manifests/e/EchoLink/EchoLink/2.3.121.1/EchoLink.EchoLink.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: EchoLink.EchoLink
+PackageVersion: 2.3.121.1
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://d34yfhzor7jtt3.cloudfront.net/EchoLinkSetup-2-3-121.exe
+  InstallerSha256: 29A5E229EACDE0495E4C977F9BB900794BDEAE907079C227BD16DEFF011314F0

```

```suggestion
- Architecture: x86
  InstallerType: burn
  InstallerUrl: https://d34yfhzor7jtt3.cloudfront.net/EchoLinkSetup-2-3-121.exe
  InstallerSha256: 29A5E229EACDE0495E4C977F9BB900794BDEAE907079C227BD16DEFF011314F0
  ProductCode: '{5ae42bc2-b60c-4a25-b0aa-c25e36aad094}'
```
Two ARP entries are written, so product code is needed to disambiguate